### PR TITLE
gesture image capture

### PR DESCRIPTION
**Pull request by proishan11**
Code to capture images for a given gesture class.

### CODE REVIEW

**Review by mohdomama — CHANGES_REQUESTED** [review]
There should be an option to add a string before image name.
Enable `anaconda` in sublime. Follow `pep8`
Move the script in `util` folder

File: CaptureImage.py [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+
+		cv2.imshow('frame', frame)
+
+		path = "./"+rootName
+
+		print(path)
+
+		cv2.imwrite(path+"/{}.jpg".format(currentFrame), frame)
+
+		if(cv2.waitKey(0) & 0xFF == ord('q')):
```
Comment (RJ722) [inline CaptureImage.py:37]: Why on earth is there a newline between every two lines?

File: CaptureImage.py [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+		if(not cap.isOpened()):
+			cap.open()
+
+		ret, frame = cap.read()
+
+		frame = cv2.resize(frame, (128, 128)) 
+
+		cv2.imshow('frame', frame)
+
+		path = "./"+rootName
```
Comment (RJ722) [inline CaptureImage.py:31]: Use `os.path.join`

File: CaptureImage.py [hunk 3]
```diff
@@ -0,0 +1,47 @@
...
+
+		print(path)
```
Comment (RJ722) [inline CaptureImage.py:33]: No need of printing the path. If you strictly need to log the path, use the `logging` module instead.

File: CaptureImage.py [hunk 4]
```diff
@@ -0,0 +1,47 @@
...
+
+		cv2.imwrite(path+"/{}.jpg".format(currentFrame), frame)
```
Comment (RJ722) [inline CaptureImage.py:35]: Again, use `os.path.join`.

File: CaptureImage.py [hunk 5]
```diff
@@ -0,0 +1,47 @@
...
+
+	cap = cv2.VideoCapture(0)
+
+	rootName = 'HandGestures/' + gestureClass
+
+	try:
+		if(not os.path.exists(rootName)):
+			os.makedirs(rootName)
+	except OSError:
+		print('Cannot create the directory')
```
Comment (RJ722) [inline CaptureImage.py:16]: Are you sure that you shall allow all the code to run, although an exception has been thrown here. You might want to exit the script here (using something like `sys.exit(1)`.

Also, change the error message to be more clear: "Directory already exists."

**Comment by RJ722** [comment]
Also, change your commit message to say something more useful than "gesture image capture"

**Comment by RJ722** [comment]
This PR was created from your master branch. You should **never** ever do that. Your master branch should always remain in sync with `upstream/master`. Always, create a new branch for different features/PR's. This will help you to:
- Revert back to master in case something goes wrong.
- Work on more than one PR's at a time.

**Comment by mohdomama** [comment]
Closing. Alternate PR will be sent for this.

**Pull request closed without merging**